python/vowpalwabbit/DFtoVWtoTensorboard.py

```diff
@@ -0,0 +1,208 @@
+from vowpalwabbit import pyvw
+from datetime import datetime
+import tensorboardX as tx
+
+
+class DFtoVWtoTensorboard:
+	"""This class provides capability for learning possible on DFtoVW object, provides capability to 
+	print metrics (average_loss, since_last, label, prediction, num_features) and also provides support 
+	to log these metrics for Tensorboard Visualization"""
+
+	def __init__(self, df_to_vw, vw):
+		"""Construct a DFtoVWtoTensorboard object
+
+		Parameters
+		----------
+
+		df_to_vw : A DFtoVW object
+					This would be used to construct examples 
+		vw		 : A vowpalwabbit object
+ 					This would be used for learning using the algorithm specified in the object, and also to compute a few metrics
+
+		Returns
+		-------
+
+		self : DFtoVWtoTensorboard
+		"""
+		self.df_to_vw = df_to_vw
```

Comment:
No need to set `df_to_vw` has instance attribute since it is not used elsewhere

Reply:
Ok gonna remove that